grafana/scylla-dash.master.template.json

```diff
@@ -258,7 +262,8 @@
                                 "step": 10
                             }
                         ],
-                        "title": "Write Timeouts per Second"
+                        "title": "Write Timeouts per Second",
+                        "description": "Requests that Scylla tried to write, but timed out. This metric is incremented in the coordinator node for the request"
```

Comment:
I don't think "This metric is incremented in the coordinator node for the request" is clear